build/document-extractor.ts

```diff
@@ -14,6 +14,7 @@ import web from "../kumascript/src/api/web";
 interface SimpleSupportStatementWithReleaseDate
   extends bcd.SimpleSupportStatement {
   release_date?: string;
+  version_supported_last?: string;
```

Comment:
To save some bytes, could we rename this property to `version_last`, or would that be ambiguous? (Since we are in the context of a support statement, I feel like "supported" is superfluous.)